tensorboard/plugins/hparams/hparams_plugin.py

```diff
@@ -149,7 +146,13 @@ def _get_scalars_plugin(self):
         scalars_metadata.PLUGIN_NAME)
 
 
-def _verify_request_is_post(request, end_point):
-  if request.method != 'POST':
-    raise error.HParamsError('%s must be a POST. Got: %s' %
-                             (end_point, request.method))
+def _parse_request_argument(request, proto_class):
```

Comment:
Is there a high level comment somewhere that mentions that both GETs and POSTs are supported, and in which cases? That would be very helpful.